samples/bluetooth/peripheral_power_profiling/README.rst

```diff
@@ -218,6 +218,11 @@ CONFIG_BT_POWER_PROFILING_NON_CONNECTABLE_ADV_INTERVAL_MIN - Non-connectable adv
 CONFIG_BT_POWER_PROFILING_NON_CONNECTABLE_ADV_INTERVAL_MAX - Non-connectable advertising maximum interval
    Sets the non-connectable advertising maximum interval in 0.625 milliseconds unit.
 
+.. _CONFIG_BT_POWER_PROFILING_LED_DISABLED:
```

Comment:
If you want to mention this new Kconfig option in the changelog, ask it in a comment to @divipillai's https://github.com/nrfconnect/sdk-nrf/pull/18192.